common/test/acceptance/tests/lms/test_learner_profile.py

```diff
@@ -641,6 +641,7 @@ def test_user_cannot_remove_default_image(self):
         self.assert_default_image_has_public_access(profile_page)
         self.assertFalse(profile_page.remove_link_present)
 
+    @flaky # TNL-3514
```

Comment:
Just a heads up you might get a pep8/pylint error here. I think you need two space characters before the comment.